website/docs/components/copy/snippet/partials/code/component-api.md

```diff
@@ -10,14 +10,17 @@ This component uses [ember-cli-clipboard](https://github.com/jkusa/ember-cli-cli
   <C.Property @name="textToCopy" @type="string" @required="true">
     String value or action that returns a string to be copied.
   </C.Property>
-  <C.Property @name="container" @type="string">
-     Selector string or element object of containing element, typically used in conjunction with modals; set the focused element as the container value.
-  </C.Property>
   <C.Property @name="isTruncated" @type="boolean" @default="false">
     Constrains text to one line and truncates it based on available width. Text will only be truncated if it does not fit within the available space.
     <br><br>
     There are accessibility concerns if using this feature. See the [Accessibility section](/components/copy/snippet?tab=accessibility) for more information.
   </C.Property>
+  <C.Property @name="onSuccess" @type="function">
+    Callback function invoked (if provided) when the "copy" action succeeds.
+  </C.Property>
+  <C.Property @name="onError" @type="function">
+    Callback function invoked (if provided) when the "copy" action fails.
+  </C.Property>
```

Comment:
Previously we decided to provide these hooks but not document them since we didn't have design considerations for them. Just wanted to make sure that we have coordinated with design before shipping this as public API

Reply:
Technically, these hooks didn't exist (check the code before), they've been added because I see a potential need to use them for the consumers (for example if they need to test some functionality that involves these components). Also, without these callback functions, is impossible to test the components too.

@heatherlarsen do you see any downsides of exposing these two callback functions (there's no impact on the component in Figma, it's a code-only thing)